src/common/metrics/src/gauge_ext.rs

```diff
@@ -22,7 +22,6 @@ pub type UintGauge = GenericGauge<AtomicU64>;
 #[easy_ext::ext(IntGaugeExt)]
 impl IntGauge {
     /// Increment the gauge, and return a guard that will decrement the gauge when dropped.
-    #[must_use]
```

Comment:
Why do we remove this?

Reply:
It's warned by rustc that `#[must_use]` has no effect when applied to a provided trait method.
<img width="676" alt="image" src="https://github.com/user-attachments/assets/3bc5007e-0959-4702-bc11-99b11717ecb2" />